go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/rackspace/gophercloud v0.0.0-20150408191457-ce0f487f6747
 	github.com/rancher/wrangler v0.8.1-0.20210506052526-673b7f8692d9
 	github.com/samalba/dockerclient v0.0.0-20151231000007-f661dd4754aa
+	github.com/sirupsen/logrus v1.4.2 // indirect
```

Comment:
machine has it's own logger, any idea how this snuck in?

Reply:
I accidentally used it without realizing that machine uses it's own logger. I will clean it up